test/parse.js

```diff
@@ -4,67 +4,67 @@ import fn from '../';
-});
-
 test('query strings starting with a `?`', t => {
-	t.context.same(fn.parse('?foo=bar'), {foo: 'bar'});
+	tsame(t, fn.parse('?foo=bar'), {foo: 'bar'});
 });
 
 test('query strings starting with a `#`', t => {
-	t.context.same(fn.parse('#foo=bar'), {foo: 'bar'});
+	tsame(t, fn.parse('#foo=bar'), {foo: 'bar'});
 });
 
 test('query strings starting with a `&`', t => {
-	t.context.same(fn.parse('&foo=bar&foo=baz'), {foo: ['bar', 'baz']});
+	tsame(t, fn.parse('&foo=bar&foo=baz'), {foo: ['bar', 'baz']});
 });
 
 test('parse a query string', t => {
-	t.context.same(fn.parse('foo=bar'), {foo: 'bar'});
+	tsame(t, fn.parse('foo=bar'), {foo: 'bar'});
 });
 
 test('parse multiple query string', t => {
-	t.context.same(fn.parse('foo=bar&key=val'), {foo: 'bar', key: 'val'});
+	tsame(t, fn.parse('foo=bar&key=val'), {foo: 'bar', key: 'val'});
 });
 
 test('parse query string without a value', t => {
-	t.context.same(fn.parse('foo'), {foo: null});
-	t.context.same(fn.parse('foo&key'), {foo: null, key: null});
-	t.context.same(fn.parse('foo=bar&key'), {foo: 'bar', key: null});
-	t.context.same(fn.parse('a&a'), {a: [null, null]});
-	t.context.same(fn.parse('a=&a'), {a: ['', null]});
+	tsame(t, fn.parse('foo'), {foo: null});
+	tsame(t, fn.parse('foo&key'), {foo: null, key: null});
+	tsame(t, fn.parse('foo=bar&key'), {foo: 'bar', key: null});
+	tsame(t, fn.parse('a&a'), {a: [null, null]});
+	tsame(t, fn.parse('a=&a'), {a: ['', null]});
 });
 
 test('return empty object if no qss can be found', t => {
-	t.context.same(fn.parse('?'), {});
-	t.context.same(fn.parse('&'), {});
-	t.context.same(fn.parse('#'), {});
-	t.context.same(fn.parse(' '), {});
+	tsame(t, fn.parse('?'), {});
+	tsame(t, fn.parse('&'), {});
+	tsame(t, fn.parse('#'), {});
+	tsame(t, fn.parse(' '), {});
 });
 
 test('handle `+` correctly', t => {
-	t.context.same(fn.parse('foo+faz=bar+baz++'), {'foo faz': 'bar baz  '});
+	tsame(t, fn.parse('foo+faz=bar+baz++'), {'foo faz': 'bar baz  '});
 });
 
 test('handle multiple of the same key', t => {
-	t.context.same(fn.parse('foo=bar&foo=baz'), {foo: ['bar', 'baz']});
+	tsame(t, fn.parse('foo=bar&foo=baz'), {foo: ['bar', 'baz']});
 });
 
 test('query strings params including embedded `=`', t => {
-	t.context.same(fn.parse('?param=http%3A%2F%2Fsomeurl%3Fid%3D2837'), {param: 'http://someurl?id=2837'});
+	tsame(t, fn.parse('?param=http%3A%2F%2Fsomeurl%3Fid%3D2837'), {param: 'http://someurl?id=2837'});
 });
 
 test('query strings params including raw `=`', t => {
-	t.context.same(fn.parse('?param=http://someurl?id=2837'), {param: 'http://someurl?id=2837'});
+	tsame(t, fn.parse('?param=http://someurl?id=2837'), {param: 'http://someurl?id=2837'});
 });
 
 test('object properties', t => {
 	t.notOk(fn.parse().prototype);
-	t.context.same(fn.parse('hasOwnProperty=foo'), {hasOwnProperty: 'fo'});
-	t.context.same(fn.parse('__proto__=bar'), {__proto__: 'bar'});
+	tsame(t, fn.parse('hasOwnProperty=foo'), {hasOwnProperty: 'foo'});
```

Comment:
Took me a while to figure out why this didn't come up in the tests. I think the expected object has to be instantiated with `Object.create(null)`.